src/httprx.js

```diff
@@ -12,12 +15,11 @@ const request = httpClient => (options, data, isBinary) => new rx.Observable(obs
             obs.complete();
         });
         resp.on('error', er => {
-            obs.error(er);
+            throw new Error(`Request failed: ${er.message}`);
```

Comment:
Here too I think this may be handled by the stream consumers, I need to look at it more.